.github/workflows/linux-publish.yml

```diff
@@ -0,0 +1,172 @@
+name: Publish Linux Packages
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
this should be running on releases, not not pushes to main

Reply:
It runs on push to main to check that the package can still be built and stops short of uploading it.

Reply:
Right. My point is "why". It seems like extra work for no value. ergo, it should only be run when making a release

Reply:
This avoids the extra work of making a new release to fix a problem that should have been caught on merge to main, if not earlier.